Gem/Code/Source/Components/NetworkWeaponsComponent.cpp

```diff
@@ -458,4 +477,10 @@ namespace MultiplayerSample
     {
         ;
     }
-}
+
+    void NetworkWeaponsComponentController::UpdateAI()
+    {
+        float deltaTime = static_cast<float>(m_updateAI.TimeInQueueMs()) / 1000.f;
```

Comment:
Our standard is using aznumeric_cast<> instead of static_cast for numbers.

Reply:
There was a discussion in another channel actually where it turns out that aznumeric_cast doesn't do anything (the assert is commented I believe)